src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -23,22 +24,22 @@ public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new EmergencyContact("87438807"),
                 new Address("Blk 30 Geylang Street 29, #06-40"), new Note(""),
-                getSubjectSet("MATH"), new Level("K2")),
+                getSubjectSet("MATH"), new Level("K2"), new TaskList()),
```

Comment:
Maybe can add something to the TaskList to one of the sample data just to visually be able to ensure that it is working properly.

Reply:
added!